src/geometry/rotate_extrude.cc

```diff
@@ -100,9 +101,8 @@ std::unique_ptr<Geometry> rotatePolygon(const RotateExtrudeNode& node, const Pol
 
   // # of sections. For closed rotations, # vertices is thus fragments*outline_size. For open
   // rotations # vertices is (fragments+1)*outline_size.
-  const auto num_sections = (unsigned int)std::ceil(fmax(
-    Calc::get_fragments_from_r(max_x - min_x, node.fn, node.fs, node.fa) * std::abs(node.angle) / 360,
-    1));
+  const int num_sections =
+    node.discretizer.getCircularSegmentCount(max_x - min_x, node.angle).value_or(5);
```

Comment:
Where did `5` come from?

Reply:
Ideally, but we just have to take care not to break any existing use-cases. ..and if we decide to break something on purpose, we should make it an isolated change and make sure to track that in release notes or smth.

Reply:
Good point. Given the context, are you alright with 5? I think 1 also makes some sense. Or to be exactly the same, `max(1,3*angle/360)`, but I hope not a lot of people set $fn="banana"...

Reply:
Well, it's truly different if $fn="banana", because it might be 5 instead of 3 or 1. So I made it exactly the same in that case, and someone can open a PR to take advantage of the optional to not generate geometry when given invalid inputs.